mlir/include/mlir/Dialect/LLVMIR/LLVMOps.td

```diff
@@ -1425,6 +1425,7 @@ def LLVM_LLVMFuncOp : LLVM_Op<"func", [
     UnitAttr:$dso_local,
     DefaultValuedAttr<CConv, "CConv::C">:$CConv,
     OptionalAttr<SymbolRefAttr>:$comdat,
+    OptionalAttr<UnitAttr>:$convergent,
```

Comment:
```suggestion
    UnitAttr:$convergent,
```
I think this is the standard way. Not sure why other `UnitAttr` are `Optional` in this operation definition.